internal/unstructured/composite.go

```diff
@@ -264,3 +265,18 @@ func (c *Composite) SetEnvironmentConfigReferences(refs []corev1.ObjectReference
 	}
 	_ = fieldpath.Pave(c.Object).SetValue("spec.environmentConfigRefs", filtered)
 }
+
+// SetObservedGeneration of this composite resource claim.
+func (c *Composite) SetObservedGeneration(generation int64) {
+	status := &xpv1.ObservedStatus{}
+	_ = fieldpath.Pave(c.Object).GetValueInto("status", status)
+	status.SetObservedGeneration(generation)
+	_ = fieldpath.Pave(c.Object).SetValue("status.observedGeneration", status.ObservedGeneration)
+}
```

Comment:
not sure where this is used? is this related to the change in a schema object?

Reply:
This is because the Composition API changed and this function is expected now.